### ISSUE TITLE
Exposure to retinoids

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9991]
**Preferred term label:**
Exposure to retinoids

**Synonyms**
Acne medications, wrinkle creams, anti-aging serum

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Telogen effluvium

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000092
- label: Retinoid exposure
- parents: [Medication history (HP:4000112)](https://hpo.jax.org/app/browse/term/HP:4000112)
- pmid: 
- def: History of having taken retinoids.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9991
- synonyms: Exposure to retinoids(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
see above

**Comment by MickeySegal** [comment]
I don't know what the "see above" refers to.

**Comment by pnrobinson** [comment]
I was not sure if you had received the first message because I forgot to tag you. I simply added the tag in the "See above".
